tests/test_crash_similarity.py

```diff
@@ -62,12 +61,11 @@ def test_order_similarity(self):
 
     def test_clean_func(self):
```

Comment:
`test_crash_similarity.py` will be removed when `crash_similarity.py` will be.  Right after I will create new classes for our models.
I just doesn't want to make useless code changes :( Nothing bad happens if our code will be in "not the best possible state" in process of big refactoring, I think.